.travis.yml

```diff
@@ -14,6 +14,6 @@ install:
     - python setup.py install
 script:
     - nosetests . --with-coverage
-    - flake8 --count pyglmnet --exit-zero;
+    - flake8 --count pyglmnet
```

Comment:
@pavanramkumar I reverted this change because we really should conform to `flake8` conventions ...